### ISSUE TITLE
Threshold on consecutive load frames

### ISSUE THREAD

**Issue by ScientiaPotestasEst** [milankarman/auto-loadless#10]
Would it be possible to add a feature where you can assign a minimum to the number of consecutive frames that have to pass the similarity check before it is counted as a load? 
This would help prevent random frames from mistakenly being counted as loads without having to up the minimum similarity too much.
In the case of the game I run/moderate we have the problem that loading screens and death screens look the same, so when using Unload we also cut out 5 frames every time the runner dies. But if we could set a threshold of say 10 frames before something is counted as a load, then this problem would be solved. 

The ability to manually remove items on the "detected loads" panel would also be nice.

**Comment by milankarman** [comment]
This sounds like a good idea. Hoping I can implement this in the next version as it doesn't seem too difficult. Do you have an example video I can test on?

Removing. editing and adding loads features are already in the works.

**Comment by ScientiaPotestasEst** [comment]
Cool to hear that editing loads is in the works.

I have 2 example videos here. In the 7th stage they abuse a mercy mechanic by killing themselves 10 times to activate a mercy checkpoint. The reason why I link 2 videos is because while the first is a shorter run and perhaps more easy to work with, I had more problems with properly detecting loads on it myself.

https://youtu.be/GiEl88Ocpns
https://youtu.be/s_Ue77tVi1w

One thing though. Our load screens are black, so it might already have some problems with Unload.
I'm still testing if using the middle of the screen as the the comparison or if I should instead use the loading icon in the bottom right (though this is an animated icon).

Thanks for your quick response.

**Comment by milankarman** [comment]
A quick update on this before I forget later. I just added the minimum consecutive frames and tested it on your second video and it seems to work quite well. As cropping I picked the animated character + version number and a bit of black space [(see image),](https://i.imgur.com/p7QYzWy.png) I set the similarity to 0.9 and minimum frames to 20.

From a quick look it seems like it picks up every load screen, however it's not perfectly at the start or end of the load screen so some manual tweaking would still be required, fortunately I'm going to release the load editing together with this so that should still be pretty easy.

Going to do some testing and add some final things and hopefully I'll be able to release it very soon.
